Formula/apache-geode.rb

```diff
@@ -7,13 +7,13 @@ class ApacheGeode < Formula
 
   bottle :unneeded
 
-  depends_on :java => "1.8+"
+  depends_on :java => "1.8"
```

Comment:
Please link to `https://issues.apache.org/jira/browse/GEODE-3` and explain that it doesn't work with Java 1.9 on the line above. Thanks!